package.json

```diff
@@ -33,7 +34,7 @@
     "start": "run-p start:css storybook",
     "start:css": "postcss src/tailwind.css -o .storybook/styles.css --watch",
     "storybook": "sleep 2 && start-storybook -p 9009",
-    "storybook:build": "npm run storybook:css && sleep 2 && build-storybook -o docs",
+    "storybook:build": "npm run storybook:css && sleep 2 && build-storybook -o docs && npm run copy:fonts",
```

Comment:
Since we are using webpack, should we use webpack to copy instead?

something like in this link:  https://www.mydatahack.com/adding-custom-font-to-storybook/

then can add in `.storybook/main.js`, so when run the build it will autocopy to the dir you specify.

Reply:
ok removed `cpy-cli` too. all use webpack copy instead

Reply:
ok